CMakeLists.txt

```diff
@@ -402,6 +402,35 @@ if (NOT MINGW AND NOT APPLE)
     set (CMAKE_EXE_LINKER_FLAGS "-Wl,-rpath=${_qt5Core_install_prefix}/lib")
     # install desktop file
     install( FILES build/mscore.desktop DESTINATION share/applications)
+    # compress man pages if gzip is present on system
+    set(MAN_TARGET ${PROJECT_SOURCE_DIR}/build/mscore.1)
+    find_program( GZIP_EXECUTABLE gzip DOC "A tool for compressing manpages (optional)." )
```

Comment:
Thanks for information, I really appreciate your help. I am still learning CMake and I am completely new to OpenBSD (and have no way of testing on it) so I didn't want to risk just taking your word on it; a link to evidence is always appreciated. The title of the page is "Getting to know OpenBSD" so it seemed appropriate to me, and I would have no way of knowing otherwise.